### PR TITLE
Improving Imports

### PR DESCRIPTION
**Pull request by fjhheras**
I am trying to understand the code. As part of my effort, I have improved the module imports, making them more explicit.

### CODE REVIEW

**Comment by fjhheras** [comment]
I am happy to change my request and explicitly import the most used
functions from .bitcoin

The main thing I intended to avoid was "from .bitcoin import *".

El miércoles, 16 de mayo de 2018, Johann Bauer <notifications@github.com>
escribió:
> @bauerj commented on this pull request.
>
> ________________________________
>
> In lib/blockchain.py:
>
>>
> MAX_TARGET =
0x00000000FFFF0000000000000000000000000000000000000000000000000000
>
> def serialize_header(res):
> - s = int_to_hex(res.get('version'), 4) \
> - + rev_hex(res.get('prev_block_hash')) \
> - + rev_hex(res.get('merkle_root')) \
> - + int_to_hex(int(res.get('timestamp')), 4) \
> - + int_to_hex(int(res.get('bits')), 4) \
> - + int_to_hex(int(res.get('nonce')), 4)
> + s = bitcoin.int_to_hex(res.get('version'), 4) \
>
> Not sure if this is an improvement. IMO this only makes the code more
verbose.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.<
https://ci3.googleusercontent.com/proxy/hTDweeNpcW5klLoHXK7jtBfH7OZlJTeHZjIaHCrEhmbsCbZm4Bl5kyDdR4jeGMuaZAVDIqGXhIsRXgU4-DPqVwHEFTVWKvBgp9qHPDV5NyagrdTa0tbFwyG3ntj7ASMiRecATPGuZ4-LM1oMY6FxbmJQTw4T7g=s0-d-e1-ft#https://github.com/notifications/beacon/ATORjAiEgSP1HNyG7JflQAFGPcZbgzCVks5tzG2wgaJpZM4UARUk.gif
>

-- 
Francisco J (Hernandez) Heras

Postdoctoral Fellow
Collective Behavior Group,
Champalimaud Neuroscience Programme

**Comment by bauerj** [comment]
Yes, replacing `*` with explicit imports sounds like a good idea! :+1:

**Comment by fjhheras** [comment]
Done

File: lib/transaction.py [hunk 1]
```diff
@@ -586,7 +591,7 @@ def multisig_script(public_keys: Sequence[str], m: int) -> str:
     assert m <= n
     op_m = format(opcodes.OP_1 + m - 1, 'x')
     op_n = format(opcodes.OP_1 + n - 1, 'x')
-    keylist = [op_push(len(k)//2) + k for k in public_keys]
+    keylist = [bitcoin.op_push(len(k)//2) + k for k in public_keys]
```
Comment (bauerj) [inline lib/transaction.py]: Why didn't you import `op_push` too?

**Comment by fjhheras** [comment]
Outdated, so I created #4448

**Pull request closed without merging**